doc/interoperability/global-scope.md

```diff
@@ -67,7 +65,7 @@ First, in any of the 4 above statements, if the referenced variable is not decla
 This is also what would happen in JavaScript when accessing a non-existent global variable.
 
 Second, whereas `val x = Globals.foo` translates to `var x = foo`, `val g = Globals` has no valid translation in JavaScript, and is a compile-time error.
-Indeed, *since ECMAScript 2015*, there is no JavaScript value that `g` could assume, such that `g.foo` would evaluate to the global variable `foo` (until ECMAScript 5.1, `g` could have been the *global object*, and this is what Scala.js 0.6.x did).
+Indeed, *since ECMAScript 2015*, there is no JavaScript value that `g` could assume, such that `g.foo` would evaluate to the global variable `foo` (until ECMAScript 5.1, `g` could have been the *global object*, and this is what earlier versions of Scala.js did).
```

Comment:
This formulation is less precise. It could refer to Scala.js 1.1.0, for example. I believe here "Scala.js 0.6.x" remains more appropriate.